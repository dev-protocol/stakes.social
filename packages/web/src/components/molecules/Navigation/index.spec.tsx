import React from 'react'
import { render } from '@testing-library/react'
import { Navigation } from './'
import 'src/__mocks__/window/matchMedia.mock'

describe(`${Navigation.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Navigation />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
