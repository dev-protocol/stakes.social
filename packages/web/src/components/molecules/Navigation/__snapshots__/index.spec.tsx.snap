// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Snapshot 1`] = `
<body>
  <div>
    <nav
      class="sc-gsDKAQ"
    >
      <div
        class="sc-bdvvtL sc-dkPtRN dGQTf kRUKIu"
      >
        <div
          class="sc-gKclnd sc-hBUSln fAyBB TXyBl"
        >
          <svgrurl
            class="sc-eCImPb bcDGrW"
            height="1.2rem"
            id="headerlogo"
          />
          <span
            class="sc-jRQBWg fpIyFp"
          >
            Stakes.social
          </span>
        </div>
        <span
          aria-label="more"
          class="anticon anticon-more sc-hKwDye jtVoEC"
          role="img"
          tabindex="-1"
        >
          <svg
            aria-hidden="true"
            data-icon="more"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M456 231a56 56 0 10112 0 56 56 0 10-112 0zm0 280a56 56 0 10112 0 56 56 0 10-112 0zm0 280a56 56 0 10112 0 56 56 0 10-112 0z"
            />
          </svg>
        </span>
      </div>
    </nav>
  </div>
</body>
`;
